### DISCUSSION TITLE
DIA data quantification - precursor level - 3-species mix

### DISCUSSION THREAD

**Discussion by mlocardpaulet** [Proteobench/ProteoBench#257] [Discuss modules in preparation]
follow up on https://github.com/orgs/Proteobench/discussions/211#discussioncomment-8619889

The idea is to create a module very similar to the DDA quant - precursor ions module, with the same raw files, using DIA runs. One of the main questions is: which raw files to use? Maybe @brvpuyve would have an idea? @wolski proposed to work on the development.

**Comment by brvpuyve** [comment]
Well, the Orbitrap data is maybe the most convenient one as it was measured on the same instrument. @wolski just let me know what you need and I can help you out with it!

**Comment by wolski** [comment]
We will be using these six files:

Condition A
[LFQ_Orbitrap_AIF_Condition_A_Sample_Alpha_01.raw](https://ftp.pride.ebi.ac.uk/pride/data/archive/2022/02/PXD028735/LFQ_Orbitrap_AIF_Condition_A_Sample_Alpha_01.raw)
[LFQ_Orbitrap_AIF_Condition_A_Sample_Alpha_02.raw](https://ftp.pride.ebi.ac.uk/pride/data/archive/2022/02/PXD028735/LFQ_Orbitrap_AIF_Condition_A_Sample_Alpha_02.raw)
[LFQ_Orbitrap_AIF_Condition_A_Sample_Alpha_03.raw](https://ftp.pride.ebi.ac.uk/pride/data/archive/2022/02/PXD028735/LFQ_Orbitrap_AIF_Condition_A_Sample_Alpha_03.raw)

Condition B
[LFQ_Orbitrap_AIF_Condition_B_Sample_Alpha_01.raw](https://ftp.pride.ebi.ac.uk/pride/data/archive/2022/02/PXD028735/LFQ_Orbitrap_AIF_Condition_B_Sample_Alpha_01.raw)
[LFQ_Orbitrap_AIF_Condition_B_Sample_Alpha_02.raw](https://ftp.pride.ebi.ac.uk/pride/data/archive/2022/02/PXD028735/LFQ_Orbitrap_AIF_Condition_B_Sample_Alpha_02.raw)
[LFQ_Orbitrap_AIF_Condition_B_Sample_Alpha_03.raw](https://ftp.pride.ebi.ac.uk/pride/data/archive/2022/02/PXD028735/LFQ_Orbitrap_AIF_Condition_B_Sample_Alpha_03.raw)

And we can use the same FASTA as before ProteoBenchFASTA_DDAQuantification.zip

**Comment by KlemensFroehlich** [comment]
I would like to propose a dual strategy of false positive assessment.
I have not yet started working on this in a systematic way so I dont want to share results just yet but here is my general thought process:

We started to look into a specific and very basic question a few months ago:
If we have a knock out sample in front of us and perform DIA then which software really gives us a "0" or missing in the KO samples and which software gives us "background" noise.

So that means for us that we have the DIA library before the main search and the entry in the lirary is real, but in the sample it is not. We could also call this matching, as it has similarities with the match between runs option in DDA.

The second part is the question whether an identification is correct during library generation / library refinement. And I think this one we can tackle with an entrapment strategy (with the limitations that Robin has discussed last time, but I still stand by my opinion that it is useful to do the entrapment search to make the false positive number or FDR more comparable between different search engines)

The reason I am bringing this up and I would love your input on this is that my initial results are indicating that Spectronaut is extremely good at the entrapment FDR control, while DIA-NN seems to be more permissve.
But when it comes to matching it looks the other way round.

SO I propose to also include files that have a "zero" spike-in, to control the maching.

I will update you on the entrapement and matching results as soon as I find time to look into this systematically!

Best Klemens

**Reply by mlocardpaulet** [reply]
Hi Klemens, thanks a lot for your contribution.
Is your idea to have a sample with many KOs? Because I don't think that we can work with only one missing protein, right? Is this even possible?
But for sure we should discuss FDP estimation in DIA. I think that this discussion would need a new discussion channel dedicated to this challenge (this one is for quantification). What do you think? Could I move your message to a new channel dedicated to a potential new module for DIA FDP estimation? Or you can totally start the new channel yourself of course :)

**Reply by KlemensFroehlich** [reply]
I see it in a slightly different way:

I think that a DIA software is reporting an identification when any value > 0 is present in the respective position of a precursor vs sample matrix.

The value that is given is the quantitation, but as soon as the value was > 0 I personally interpret this as an "identification" via the software.

This is why I would already use both approaches in the identification, not only the quantitation.

My suggestion would be to use a sample with a spike-in and another sample where this spike-in did not occur. For example: we used a E.coli in HEK spike-in in our last publication. I am of course happy to use any other dataset, but I think it is fairly high quality and would give us some IDs to work with while the E.coli ratio is so small that there is virtually no displacement happening with the HEK abundances.


![image](https://github.com/Proteobench/ProteoBench/assets/22170937/fb593d06-32e8-4581-b68c-00b08712e9c4)

These are 2 x 9 samples: 2x ( 3 without Ecoli, 3 with 1:20 and 3 with 1:10 Ecoli/HEK ) 
The different bars indicate different measurement and software parameters. 

We could simply choose one of those measurement options so that we end up with 1 x 9 samples.

doi/10.1021/acs.jproteome.3c00736
How would you separate the ID from the quantitation when thinking about benchmarking:
- spectral library FDR
- identification per run using said spectral library
- quantitation per run using said library

If I udnerstand you correctly you would prefer to keep the two last points associated with quantitation?

Best, Klemens

**Reply by vdemichev** [reply]
> my initial results are indicating that Spectronaut is extremely good at the entrapment FDR control

Just a comment, that would appear to be the case if either (i) directDIA is used in regular, not 'method evaluation' mode - i.e. results are with MBR which may signficantly decrease run-specific FDR as estimated with entrapment or (ii) any protein-level q-value filtering is applied, in which case precursor-level entrapment FDR estimates can (falsely) go down 5x-10x.

**Comment by mlocardpaulet** [comment]
we are working on it: https://github.com/Proteobench/ProteoBench/issues/348
